Marlin/ultralcd.cpp

```diff
@@ -836,7 +836,9 @@ void kill_screen(const char* lcd_msg) {
 
     void _lcd_user_gcode(const char * const cmd) {
       enqueue_and_echo_commands_P(cmd);
-      lcd_completion_feedback();
+      #ifdef USER_SCRIPT_FEEDBACK
```

Comment:
In other words, use `ENABLED` for simple on/off config options, and use `#ifdef` or `defined` for options taking a value. You can also use plain old `THING > 0` or `THING != 0` in cases where it may be more clear, because in preprocessor expressions undefined symbols evaluate to 0.